react-exercises/modal-react/src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './App.css';
+import Modal from  './Modal.js';
+
+
+function App() {
+  return (
+    <div className="App">
+      <Modal login='true' signup="true"/>
```

Comment:
Remember what we are creating is a generic Modal component. Login and Sign up are just few use cases. Your modal should be able to take child components.